core/domain/activity_jobs_one_off_test.py

```diff
@@ -2078,3 +2081,412 @@ def test_audit_multiple_rights_snapshots(self):
                 ['topic-length-0', 1]
             ]
         )
+
+
+class ValidateSnapshotMetadataModelsJobTests(test_utils.GenericTestBase):
+    ALBERT_EMAIL = 'albert@example.com'
+    ALBERT_NAME = 'albert'
+
+    EXP_ID = 'exp_id0'
+    COLLECTION_ID = 'collection_id0'
+    QUESTION_ID = 'question_id0'
+    SKILL_ID = 'skill_id0'
+    STORY_ID = 'story_id0'
+    TOPIC_ID = 'topic_id0'
+    # The subtopic snapshot ID is in the format
+    # '<topicId>-<subtopicNum>-<version>'.
+    SUBTOPIC_ID = 'topic_id0-1'
+    TOPIC_RIGHTS_ID = 'topic_rights_id0'
+    DUMMY_COMMIT_CMDS = [
+        {
+            'cmd': 'some_command',
+            'other_field': 'test'
+        }, {
+            'cmd': 'some_other_command',
+            'other_field': 'test',
+            'different_field': 'test'
+        }
+    ]
+
+    def setUp(self):
+        super(ValidateSnapshotMetadataModelsJobTests, self).setUp()
+
+        # Setup user who will own the test explorations.
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+        self.process_and_flush_pending_tasks()
+
+    def _run_one_off_job(self):
+        """Runs the one-off MapReduce job."""
+        job_id = (
+            activity_jobs_one_off.ValidateSnapshotMetadataModelsJob.create_new()) # pylint: disable=line-too-long
```

Comment:
Don't do this, make a local variable `job_class = activity_jobs_one_off.ValidateSnapshotMetadataModelsJob`

In general, don't use pylint pragmas. Avoid them as much as you can.

Reply:
Done.